lib/service/logger.js

```diff
@@ -24,7 +24,7 @@ function _log(level, message) {
     level
   };
 
-  logMessage = JSON.stringify(logMessage) + '\n';
+  logMessage = `${JSON.stringify(logMessage)}\n`;
```

Comment:
it's not obvious how nodejs implements this internally.  Runtimes are mostly similar (though for some patterns string addition is faster).

Looping them, generating new distinct strings grows the heap with both but interpolation seems to be putting more strain on the heap (more gc activity, more memory used).  Re-generating a previously created string does not grow the heap (string reuse ?!)

Over 200k operations string addition uses 8mb less heap:
```
% time node --trace-gc -p 'var x, i, a=1, b=2; for (i=0; i<200000; i++) x = i + a + b + "\n"; process.memoryUsage()'
[16370:0xa8e4780]        5 ms: Scavenge 1.6 (17.5) -> 1.6 (17.5) MB, 0.7 / 0 ms [allocation failure].
[16370:0xa8e4780]        8 ms: Scavenge 1.7 (17.5) -> 1.7 (18.5) MB, 0.7 / 0 ms [allocation failure].
[16370:0xa8e4780]       45 ms: Scavenge 3.5 (21.5) -> 2.9 (21.5) MB, 0.5 / 0 ms [allocation failure].
[16370:0xa8e4780]       58 ms: Scavenge 6.0 (23.5) -> 4.4 (23.5) MB, 0.7 / 0 ms [allocation failure].
[16370:0xa8e4780]       75 ms: Scavenge 8.3 (25.5) -> 6.4 (25.5) MB, 0.1 / 0 ms [allocation failure].
{ rss: 26722304, heapTotal: 13537280, heapUsed: 6713836 }
0.096u 0.016s 0:00.11 90.9%	0+0k 0+0io 0pf+0w
# using 6.7m, gc'd 4.1m

% time node --trace-gc -p 'var x, i, a=1, b=2; for (i=0; i<200000; i++) x = `${i}${a}${b}\n`; process.memoryUsage()'
[16376:0x97bf780]        6 ms: Scavenge 1.6 (17.5) -> 1.6 (17.5) MB, 0.8 / 0 ms [allocation failure].
[16376:0x97bf780]        8 ms: Scavenge 1.7 (17.5) -> 1.7 (18.5) MB, 0.8 / 0 ms [allocation failure].
[16376:0x97bf780]       42 ms: Scavenge 3.2 (21.5) -> 2.7 (21.5) MB, 0.6 / 0 ms [allocation failure].
[16376:0x97bf780]       50 ms: Scavenge 4.7 (21.5) -> 3.2 (22.5) MB, 0.7 / 0 ms [allocation failure].
[16376:0x97bf780]       58 ms: Scavenge 5.8 (22.5) -> 3.8 (22.5) MB, 0.1 / 0 ms [allocation failure].
[16376:0x97bf780]       65 ms: Scavenge 6.4 (23.5) -> 4.5 (23.5) MB, 0.1 / 0 ms [allocation failure].
[16376:0x97bf780]       72 ms: Scavenge 7.0 (24.5) -> 5.1 (24.5) MB, 0.2 / 0 ms [allocation failure].
[16376:0x97bf780]       79 ms: Scavenge 7.6 (24.5) -> 5.7 (24.5) MB, 0.2 / 0 ms [allocation failure].
[16376:0x97bf780]       86 ms: Scavenge 8.2 (25.5) -> 6.3 (25.5) MB, 0.2 / 0 ms [allocation failure].
{ rss: 26681344, heapTotal: 13537280, heapUsed: 7087904 }
0.112u 0.012s 0:00.12 100.0%	0+0k 0+0io 0pf+0w
# using 7.1m, gc'd 11.6m
```